src/test/java/duke/TodoTest.java

```diff
@@ -0,0 +1,15 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TodoTest {
+    @Test
+    public void testTodoStringConversion(){
```

Comment:
In your TP, please pay attention to the test names; our coding standards specify 3 part naming for test methods.